.markdownlint.yml

```diff
@@ -0,0 +1,19 @@
+# SPDX-FileCopyrightText: Copyright (c) 2012-2025 Yegor Bugayenko
```

Comment:
@h1alexbel why not fixing them all in the MD files? They look like reasonable complaints.

Reply:
@yegor256 we can't fix `MD033`, since we need specify `width` and `height` in the `img` of JCabi image

Reply:
@h1alexbel we can specify them inside the SVG file. Also, we can simply remove the log from the README.

Reply:
@yegor256 didn't know about SVG sizes, many thanks